### PR TITLE
Images/git: Build for ppc64le

### PR DESCRIPTION
**Pull request by deepsm007**
This commit adds a Google Cloud Build definition for building git
ppc64le images and pushing them as a manifest list together with their
ppc64le counterparts.

Blocked on: #23276

/cc @mkumatag 
/cc @stevekuznetsov

rel: #16588

### CODE REVIEW

**Comment by mkumatag** [comment]
/lgtm
/hold
/ok-to-test
hold until #23276 merges

**Review by alvaroaleman — COMMENTED** [review]
/assign @chaodaiG

**Review by chaodaiG — APPROVED** [review]

**Comment by mkumatag** [comment]
@deepsm007 please squash the commits

**Review by mkumatag — APPROVED** [review]
/lgtm
/unhold

**Comment by mkumatag** [comment]
now we have a new image with ppc64le

```shell
$ manifest-tool inspect gcr.io/k8s-prow/git:latest
Name:   gcr.io/k8s-prow/git:latest (Type: application/vnd.docker.distribution.manifest.list.v2+json)
Digest: sha256:dfe311e5d8f01035654611895bd3be43e28253cf96d2c3ab4bb0a795fa9d5a63
 * Contains 3 manifest references:
1    Mfst Type: application/vnd.docker.distribution.manifest.v2+json
1       Digest: sha256:3fef8233c73aa85f953f0f90a3dc0daee88b2eea5210af0779512c34ce5f1ae6
1  Mfst Length: 949
1     Platform:
1           -      OS: linux
1           - OS Vers:
1           - OS Feat: []
1           -    Arch: amd64
1           - Variant:
1           - Feature:
1     # Layers: 3
         layer 1: digest = sha256:29291e31a76a7e560b9b7ad3cada56e8c18d50a96cca8a2573e4f4689d7aca77
         layer 2: digest = sha256:d8b0fef36ee8776d54c12fed24e6eae5e3ad0253ca18dca4e57e90f1d5c85eda
         layer 3: digest = sha256:2ca59ef288e1fcd8e2d6b3d4eb17807adfb38d42a4b69f49a59be159eb86d119

2    Mfst Type: application/vnd.docker.distribution.manifest.v2+json
2       Digest: sha256:17373d3456b4ad146b42b23fd4b6322dc804b62af6c3f939fbf7533239d2dce5
2  Mfst Length: 949
2     Platform:
2           -      OS: linux
2           - OS Vers:
2           - OS Feat: []
2           -    Arch: arm64
2           - Variant:
2           - Feature:
2     # Layers: 3
         layer 1: digest = sha256:fd3acdcea5682abced546ec19fb6ebee725c5184e5d91614c469c0a79e67f2d0
         layer 2: digest = sha256:c870bdb2572e9e2969fa459f01b8acf3a69083113b44c807043920679525b54a
         layer 3: digest = sha256:e0ed68c040143e3d054c685324ed2f5adf1ac650c19dbf172beedd0b920fbc88

3    Mfst Type: application/vnd.docker.distribution.manifest.v2+json
3       Digest: sha256:cc5b28dfc4165dccc91d0cf04e9addf1b2d22ab70b9a5e313b6299a4aa1d6646
3  Mfst Length: 949
3     Platform:
3           -      OS: linux
3           - OS Vers:
3           - OS Feat: []
3           -    Arch: ppc64le
3           - Variant:
3           - Feature:
3     # Layers: 3
         layer 1: digest = sha256:0ff902055236f70c4694c806877243e1dd52c513825a2a3ecc7eba8f5202acc8
         layer 2: digest = sha256:6b6905b8b6cc973d62fcdf70a92e8ef588196f2c5ae9da3a0551528324419683
         layer 3: digest = sha256:f89ce5ebaa0310ee79dd8ff0dea5e2c2b7664c85824ac531ba862bb40707af8f
```

**Pull request merged**
